### ISSUE TITLE
Add way to remove `signal` generation.

### ISSUE THREAD

**Issue by arthurfiorette** [anymaniax/orval#395]
We're using orval with some custom functions and we need to know the `options?: SecondParameter<typeof customInstance>` parameter index.

As orval is a generator, except when it is a `get` method, the signal parameter (which, if removed, can be used inside the options parameter), alters the order of them.

Could you add an option to opt-out of the #357 feature?

**Comment by aodinok** [comment]
We also have issue with signal generation on some of our methods. 

In some cases generated query function for axios doesn't have signal parameter but hook that is trying to call it passing signal - which is causing error about extra parameter passed to function that has only 3 arguments

**Comment by anymaniax** [comment]
Will it solve also your problem if I pass it like this

```ts
export const listPets = (
  params?: ListPetsParams,
  version = 1,
  signal?: AbortSignal,
) => {
  return customInstance<Pets>({
    url: `/v${version}/pets`,
    method: 'get',
    ...(params ? { params } : {}),
    ...(signal ? { signal } : {}),
  });
};

```

**Comment by dhess** [comment]
Honestly, I'm not sure &mdash; I'm often surprised by what TypeScript complains about with optionals :)

**Comment by anymaniax** [comment]
@dhess for the typescript issue I tested and it seems to work like that since I don't pass the prop

**Comment by arthurfiorette** [comment]
I still think that a opt-out option should be added...

Our issue (the reason I opened #395) will only be resolved when the `signal` option gets removed. Leaving the "overriding options" always as the last parameter.

**Comment by anymaniax** [comment]
I will also add this option

**Comment by anymaniax** [comment]
signal will now only be added for `react-query`, `vue-query` & `svelte-query` and you would be able to override the generation of signal like [this](https://orval.dev/reference/configuration/output#signal). Will be in the next release

**Comment by anymaniax** [comment]
Released with 6.9.0
